CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 
 ### Fixed
 
+- [#8399](https://github.com/thanos-io/thanos/pull/8399) Thanos quickstart.sh: Fix ObjStore config lookup for non-MinIO use
```

Comment:
I don't think quickstart should be part of release notes, it's not relevant for Thanos users (only devs), no?

Reply:
Done!